libs/langchain-mongodb/langchain_mongodb/vectorstores.py

```diff
@@ -99,20 +99,11 @@ class MongoDBAtlasVectorSearch(VectorStore):
             from pymongo import MongoClient
             from langchain_openai import OpenAIEmbeddings
 
-            # initialize MongoDB python client
-            client = MongoClient(MONGODB_ATLAS_CLUSTER_URI)
-
-            DB_NAME = "langchain_test_db"
-            COLLECTION_NAME = "langchain_test_vectorstores"
-            ATLAS_VECTOR_SEARCH_INDEX_NAME = "langchain-test-index-vectorstores"
-
-            MONGODB_COLLECTION = client[DB_NAME][COLLECTION_NAME]
-
-            vector_store = MongoDBAtlasVectorSearch(
-                collection=MONGODB_COLLECTION,
+            vector_store = MongoDBAtlasVectorSearch.from_connection_string(
+                connection_string="mongodb+srv://user:password@clusterip.mongodb.net/?w=majority&appName=Cluster0"
```

Comment:
Is it safe, to make this easier on the eyes, to remove all of the metadata in the connection_string? So it's just 
`connection_string="mongodb+srv://user:password@clusterip.mongodb.net`?

Reply:
I think keeping the parts in that are part of the standard Atlas connection string makes sense

Reply:
Can we keep it to MONGODB_ATLAS_CLUSTER_URI